pkg/postgres/pgutils/error.go

```diff
@@ -36,7 +36,9 @@ var transientPGCodes = set.NewFrozenStringSet(
 
 	// Class 57 — Operator Intervention
 	"57000", // operator_intervention
-	"57014", // query_canceled
+
+	// "57014", // query_canceled - this means there is a statement timeout and retrying will most likely not result in greater success
```

Comment:
How do you tell that "retrying will most likely not result in greater success"?

Reply:
Yeah, this might be a overstatement. It's unlikely, but still possible if there was a high load on the DB then the query may be successful after the initial part